docs/src/main/tut/freemonad.md

```diff
@@ -166,14 +166,14 @@ DSL. By itself, this DSL only represents a sequence of operations
 To do this, we will use a *natural transformation* between type
 containers.  Natural transformations go between types like `F[_]` and
 `G[_]` (this particular transformation would be written as
-`NaturalTransformation[F,G]` or as done here using the symbolic
+`FunctionK[F,G]` or as done here using the symbolic
 alternative as `F ~> G`).
```

Comment:
This could then be something like : 

> To do this, we will use  a `FunctionK` (also known as a natural transformation). A `FunctionK` is like a function but between type containers instead of values. A `FunctionK` between types `F[_]` and `G[_]` could be written as `FunctionK[F, G]` (or as `F ~> G` using the symbolic alternative).